persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/ConversationDAOImpl.kt

```diff
@@ -101,6 +101,17 @@ class ConversationDAOImpl(
 
     }
 
+    override suspend fun insertOrUpdateOneOneMemberWithConnectionStatus(
```

Comment:
I don't know for sure, but is this a typo ? Or it has to do with `OneToOneConversations` ? I think we should fix it accordingly; to `OneToOne` or removing one of the `Ones` 🤔  ? (no pun intended xd)

Reply:
> I don't know for sure, but is this a typo ? Or it has to do with `OneToOneConversations` ? I think we should fix it accordingly; to `OneToOne` or removing one of the `Ones` 🤔 ? (no pun intended xd)

Yeah.

We have some places saying `OneOnOne` and some places saying `OneOne` already.

I f*cked up when I introduced `OneOne` to the project. I think `OneOnOne` or `OneToOne` are clearer and less prone to misunderstanding.

I suggest using `OneOnOne`, as it's already in the project.

We can erradicate `OneOne` from other parts later.

Reply:
Changing to `OneOnOne` then :)

Reply:
FYI have also some places saying `OneToOne`.